app/views/unattended/provisioning_templates/snippet/kickstart_kernel_options.erb

```diff
@@ -157,6 +160,15 @@ description: |
   elsif nic_delay
     options.push("nicdelay=#{nic_delay} linksleep=#{nic_delay}")
   end
+
+  # add user specified custom options
+  if custom_options.length > 0 
+    if custom_options.respond_to?('join')
+      options.push(custom_options.join(' '))
```

Comment:
You can also use `options += custom_options` or `options.concat(custom_options)` if it's an array.